daemon/graphdriver/quota/projectquota.go

```diff
@@ -38,15 +38,31 @@ struct fsxattr {
 #ifndef PRJQUOTA
 #define PRJQUOTA	2
 #endif
-#ifndef XFS_PROJ_QUOTA
-#define XFS_PROJ_QUOTA	2
+#ifndef PROJ_QUOTA
```

Comment:
That ifdef was added to fix build issues on some systems. Not sure you can remove or change it